lib/reply.js

```diff
@@ -508,7 +496,7 @@ function sendStream (payload, res, reply) {
 }
 
 function onErrorHook (reply, error, cb) {
-  if (reply.context.onError !== null && reply[kReplyErrorHandlerCalled] === true) {
+  if (reply.context.onError !== null && !reply[kReplyNextErrorHandler]) {
```

Comment:
Is the coercion check necessary? Or can it be `=== false`?

Reply:
The coercion is necessary.  There might be a bug hiding somewhere around this check as it was one of the most troubling spot while doing this change.

Reply:
Let's add a comment to that effect.